target/linux/ath79/dts/ar9344_devolo_magic-2-wifi.dts

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,magic-2-wifi", "qca,ar9344";
+
+	aliases {
+		led-boot = &plcr;
+		led-failsafe = &plcr;
+		led-running = &plcw;
+		led-upgrade = &plcr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan {
+			label = "devolo:status:wlan";
```

Comment:
This is supposed to be "device:color:usage", so "devolo:yourledscolor:wlan". Same for the LEDs below.
The node name is then supposed to have the name after the last colon, so in this case wlan (which you did correctly). If you then want to have a LED label, like for use in aliases, you would prefix that by a DTS label "led_wlan: ". Please do so accordingly for the LEDs below.